yt_dlp/YoutubeDL.py

```diff
@@ -4339,6 +4330,7 @@ def _write_thumbnails(self, label, info_dict, filename, thumb_filename_base=None
                 self.to_screen(f'[info] There are no {label} thumbnails to download')
                 return ret
         multiple = write_all and len(thumbnails) > 1
+        info_dict['__multiple_thumbnails'] = multiple
```

Comment:
This is how I'm determining whether thumbnails need to use an indexed file "extension". I _could_ use `get_param('write_all_thumbnails')` and similar in the `MoveFilesPP`, but I reckon doing it this way lessens the risk of logic here getting out of sync with logic in `MoveFilesPP` if this is refactored/modified in the future